### PR TITLE
INS-13147

### PR DESCRIPTION
**Pull request by Chrismelba-Instaclustr**
Updated documentation for Terraform Registry

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -5,19 +5,67 @@ description: |-
...
-## Schema
+Use the navigation to the left to read about the available resources.
 
-### Required
 
-- **api_key** (String)
-- **username** (String)
+### Example Usage
+
+```
```
Comment (iamhail) [inline docs/index.md:23]:
```tf for syntax highlighting

File: docs/resources/cluster.md [hunk 2]
```diff
@@ -7,59 +7,91 @@ description: |-
...
+private_network_cluster|Accepts true/false. Creates the cluster with private network only.|false
+pci_compliant_cluster|Accepts true/false. Creates the cluster with PCI compliance enabled.|false
+cluster_provider|The information of infrastructure provider. See below for its properties.|Required
+rack_allocation|The number of resources to use. See below for its properties.|Optional, but Required for all Bundle types excluding Redis.
+bundle|Array of bundle information. See below for its properties.|Required
+kafka_rest_proxy_user_password|The password of kafka rest proxy bundle user, if it is a Kafka cluster with rest-proxy addon. This field is updatable and requires `wait_for_state` to be `RUNNING`.|Optional
+kafka_schema_registry_user_password|The password of kafka schema registry bundle user, if it is a Kafka cluster with schema-registry addon. This field is updatable and requires `wait_for_state` to be `RUNNING`.|Optional
+wait_for_state|The expected state of the cluster before completing the resource creation. Skipping this field will asynchronously create the cluster.|Optional (valid states are RUNNING and PROVISIONED)
+
+`cluster_provider`
```
Comment (iamhail) [inline docs/resources/cluster.md:30]: Can we make this a sub heading? ### i think

File: docs/resources/kafka_user.md [hunk 3]
```diff
@@ -5,23 +5,42 @@ description: |-
...
-- **password** (String, Sensitive)
-- **username** (String)
-
-### Optional
-
-- **id** (String) The ID of this resource.
-- **initial_permissions** (String)
-
-
+# Resource  `instaclustr_kafka_user` and Data Source `instaclustr_kafka_user_list`
```
Comment (iamhail) [inline docs/resources/kafka_user.md:8]: Can we separate the Data Source in another hierarchy (docs/data-sources)?
https://www.terraform.io/docs/registry/providers/docs.html#directory-structure

Comment (Chrismelba-Instaclustr) [reply docs/resources/kafka_user.md:8]: Done, though I'm not entirely sure how this will return to make an example?

**Review by peter-toms — APPROVED** [review]

**Review by iamhail — APPROVED** [review]

**Review by hail-insta — APPROVED** [review]

**Pull request merged**